.github/workflows/test.yml

```diff
@@ -485,13 +485,16 @@ jobs:
     - name: Install Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: '16'
+          node-version: '18'
     - run: npm install
       working-directory: ./vscode
     - run: npm run compile
       working-directory: ./vscode
     - name: Run headless test
-      run: xvfb-run npm test
+      run: |
+          export DBUS_SESSION_BUS_ADDRESS=/dev/null
+          export NO_AT_BRIDGE=1
+          xvfb-run --auto-servernum --server-args="-screen 0 1920x1080x24" npm test -- --disable-gpu
```

Comment:
These changes don't fix the issue of the vscode test not working. For some reason the tap package is not installed, it's not `xvfb-run`

Reply:
Let me just remove that and address it in a further PR then